content/en/docs/chart_template_guide/function_list.md

```diff
@@ -1458,6 +1460,7 @@ newdict:
   default: default
   overwrite: overwritten
   key: false
+  nullify: nil # AKA null
 ```
```

Comment:
I think adding this to the main example is just going to cause confusion for most people that are not concerned with this case. I think adding some prose down further along the lines of "Overriding a dictionary with a `nil` value does not delete the key value pair, it will not result in a dictionary with a key to a `nil` value"

but IMNAM